entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/ZohoSyncService.java

```diff
@@ -503,6 +549,8 @@ private void addOperation(
 
```

Comment:
The use of the buildEntityIdUri method needs to be replaced, as the value is now taken from the "Europeana ID" field